constants/item_constants.asm

```diff
@@ -52,13 +52,13 @@ DEF SAFARI_ROCK EQU $16 ; overload
 	const DOME_FOSSIL   ; $29
 	const HELIX_FOSSIL  ; $2A
 	const SECRET_KEY    ; $2B
-	const UNUSED_ITEM   ; $2C "?????"
+	const DUMMY_ITEM    ; $2C ; unused
```

Comment:
What's the motivation for this change? I'm neither for or against the new name hypothetically, but again why introduce renames that will throw off old code if there isn't a compelling reason?

Reply:
I don't mean the comment, I mean the constant. Why is `DUMMY_ITEM` better than `UNUSED_ITEM`?

Reply:
That seems like a poor reason to change the name.

Reply:
`UNUSED_ITEM` is good and doesn't need changing. We already use "UNUSED" in many other constant names, like all the unused maps and unused sprites, as well as a couple others.